specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/OperationsList.json

```diff
@@ -4,2267 +4,2270 @@
+                  }
+               },
+               {
+                  "name":"Microsoft.OperationalInsights/workspaces/search/action",
+                  "display":{
+                     "provider":"Microsoft Operational Insights",
+                     "resource":"Search",
+                     "operation":"Search Workspace Data",
+                     "description":"Executes a search query"
+                  }
+               },
+               {
+                  "name":"Microsoft.OperationalInsights/workspaces/schema/read",
+                  "display":{
+                     "provider":"Microsoft Operational Insights",
+                     "resource":"Search Schema",
+                     "operation":"Get Search Schema",
+                     "description":"Gets the search schema for the workspace.  Search schema includes the exposed fields and their types."
+                  }
+               },
+               {
+                  "name":"Microsoft.OperationalInsights/workspaces/datasources/read",
+                  "display":{
+                     "provider":"Microsoft Operational Insights",
+                     "resource":"Data Source",
+                     "operation":"Get datasources under a workspace.",
+                     "description":"Get datasources under a workspace."
+                  }
+               },
+               {
+                  "name":"Microsoft.OperationalInsights/workspaces/datasources/write",
+                  "display":{
+                     "provider":"Microsoft Operational Insights",
+                     "resource":"Data Source",
+                     "operation":"Create/Update datasources under a workspace.",
+                     "description":"Create/Update datasources under a workspace."
+                  }
+               },
+               {
 
-            },
-            {
+               },
+               {
 
-            },
-            {
+               },
+               {
 
-            },
-            {
+               },
+               {
 
-            },
-            {
-               "name":"Microsoft.OperationalInsights/workspaces/notificationSettings/read",
-               "display":{
-                  "provider":"Microsoft Operational Insights",
-                  "resource":"Notification Settings",
-                  "operation":"Get Notification Settings",
-                  "description":"Get the user's notification settings for the workspace."
-               }
-            },
-            {
-               "name":"Microsoft.OperationalInsights/workspaces/notificationSettings/write",
-               "display":{
-                  "provider":"Microsoft Operational Insights",
-                  "resource":"Notification Settings",
-                  "operation":"Put Notification Settings",
-                  "description":"Set the user's notification settings for the workspace."
-               }
-            },
-            {
+               },
```

Comment:
Seems like some formatting got messed up in the most recent iteration. Mind moving back to two-space indentation?

Reply:
Look at the beginning of the file, there is now a "value" node, that indent everything further ;)

Reply:
Whoops, nevermind then!